combination_test.go

```diff
@@ -0,0 +1,63 @@
+package errors
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures Do not use asserts. Use useful test failures.